frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -63,8 +63,11 @@ export class Wizard {
       await click(wizardView.cloudInitCustomScriptCheckbox);
       await fillInput(wizardView.customCloudInitScriptTextArea, cloudInitOptions.customScript);
     } else {
```

Comment:
this will not work when selecting a template with customScript (will switch to Custom script automatically). 
Can we first check for the state of the radio buttons and click accordingly? Or just click at it unconditionally?

Reply:
we can also get a confirmation window. We should probably test that as well. Maybe in the followup?

Reply:
Indeed, this doesn't account for the case of creating VM from a template.
I incline to extending this logic in a followup as it is not required atm.

Reply:
ok